src/main/java/com/hillel/items_exchange/exception/handler/GlobalExceptionHandler.java

```diff
@@ -121,6 +122,14 @@
         return new ResponseEntity<>(errorMessage, HttpStatus.BAD_REQUEST);
     }
 
+    @ExceptionHandler(AccessDeniedException.class)
+    public ResponseEntity<String> handlerAccessDenied(AccessDeniedException e) {
```

Comment:
Would you mind to refactor this handler to make it look like the rest? You can use ErrorMassage, extract `.body(...)` to property file and log error with `logErrorMessage()`

Reply:
@KozminAlexandr I've just moved the code to current place instead from the controller. Current ticket blocks 2 other tickets that @lubchik-14 and @drw85 are working on.
If you think that it's on time and you can do it very quickly - go ahead!

Reply:
@rpkyrych If you do not mind, I will do it in my next ticket